providers/gce/gce.go

```diff
@@ -195,6 +195,13 @@ type Cloud struct {
 	// stackType indicates whether the cluster is a single stack IPv4, single
 	// stack IPv6 or a dual stack cluster
 	stackType StackType
+
+	// multiProject determines whether the project derived through the Node's
+	// .spec.providerID can be used to change the project used when making GCE
+	// API calls.
+	//
+	// Enable this ony when the Node's .spec.providerID can be fully trusted.
+	multiProject bool
```

Comment:
A bit of a naming thing -- multiproject is the outer feature, but this sounds like it should be called `projectFromNodeProviderID` or something.

Reply:
Sounds reasonable and helps with clarity. Incorporated, thanks!